### DISCUSSION TITLE
do we have to upgrade lily to v0.15.0 before the 27th?

### DISCUSSION THREAD

**Discussion by wuqinqiang** [filecoin-project/lotus#10751] [Build and Sync] [answered]
The main network is going to be strongly updated to lotusv1.22.0 on the 27th,I don't see any version of lily about lotusv1.22.0,do we have to upgrade lily to v0.15.0 before the 27th?

**Comment by rjan90** [comment, marked as answer]
Sentinel have released Lily v0.15.0 which support the upcoming nv19 & nv20 upgrade:
https://github.com/filecoin-project/lily/releases/tag/v0.15.0

**Reply by wuqinqiang** [reply]
thanks~~
